### PR TITLE
Add Azure deploy script and webjob

### PR DESCRIPTION
**Pull request by tvararu**
Once this hits master on Travis, hopefully it should deploy as a webjob on the `search-and-compare-ui` project.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,3 +19,13 @@ notifications:
     secure: L4jZaDmU/D4koIjWfL1D6vsc6eh6N4Qlhwgdf3ymjwK7KB2e66ekxIeuMvBRIWtu8bYnpdkqT+RSR2dNW6i0Qw78stDYy7i118Zxbnd3xlLOK6ClXElYFIcoMaFVlOS1wGRLH++0QP4WfQx8YY7qKrH0RLkmZeTiYDT83G7Mg+Inuf4A8adh51fGtOjzMJ86nugkEFpkad+t/yQI7zOb/bwyoYCr0JR1GmgcBUssgJq+MhjmBJd7deATIiIUCpm3kMBqWyfBkS++pnb4d/9AkMZqTajiYV6x121YUU2Nz5Avf8xGJgLOJP9eycNr7VkLePJv7w/a0+YIissVXLMbj2xJNunThy+5vV+Pp6ifEK/lL7eGcvY1u5wvqKuzO226WFSIq7A4pwbB+3dFv1SYtXqvv67Wpi4i0C5jkVauFIBpzXKZfyBsy3WTTq7AX0Seu6rtSpC/jGbuZIo6GBQnvEjWsl0raqDb9gh+/g6WG898tawlVcUiKsixitYODpWG5kdbB5AbJYjTY24krZeySj7pMKMzsXJJ7ZSCUKtJBpieFqo1bu3DMI1m7ZDDVwA2y8o1n8FsLAwLI17By5huiACvy7kQnahecycUKr8o7yZdTmjUKTxNJ5lxKsS1bTYjjGPOj7DufxTEUTGRVgMkMy+z2/tCiYCL7Z2+92jNv6c=
   on_success: change
   on_failure: always
+deploy:
+  - provider: script
+    script: "./travis-deploy-azure.sh govuk-dfe-bat-deployment-user:$AZURE_WA_PASSWORD bat-dev-search-and-compare-ui-app"
```
Comment (the-daniel-rothig) [inline .travis.yml]: is $AZURE_WA_PASSWORD set via `travis encrypt`? Just asking b/c I was expecting a new `secure` in the diff for this (unless the `secure` immediately above is AZURE_WA_PASSWORD)

File: settings.job [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "schedule": "0 */15 * * * *"
```
Comment (the-daniel-rothig) [inline settings.job]: should be light enough for every 5 minutes imo?

File: .travis.yml (hunk 1 above)
Comment (tvararu) [reply .travis.yml]: I've set it via the user interface at https://travis-ci.org/DFE-Digital/search-and-compare-ui-tests/settings, basically accomplishes the same thing.

File: settings.job (hunk 2 above)
Comment (tvararu) [reply settings.job]: They currently run in ~20 seconds (excluding the time to spin up the node deps and build cache on travis) so yes they could!

**Pull request merged**